source/jobs.c

```diff
@@ -72,6 +72,47 @@ static const size_t apiTopicLength[] =
     JOBS_API_UPDATE_LENGTH + JOBS_API_FAILURE_LENGTH,
 };
 
+static const char * const jobStatusString[ 5U ] =
+{
+    "QUEUED",
+    "IN_PROGRESS",
+    "FAILED",
+    "SUCCEEDED",
+    "REJECTED"
+};
+
+static const size_t jobStatusStringLengths[ 5U ] =
+{
+    sizeof( "QUEUED" ) - 1U,
+    sizeof( "IN_PROGRESS" ) - 1U,
+    sizeof( "FAILED" ) - 1U,
+    sizeof( "SUCCEEDED" ) - 1U,
+    sizeof( "REJECTED" ) - 1U
+};
+
+static const char * const jobUpdateStatusString[ 2U ] =
+{
+    "accepted",
+    "rejected"
+};
+
+static const size_t jobUpdateStatusStringLengths[ 2U ] =
+{
+    sizeof( "accepted" ) - 1U,
+    sizeof( "rejected" ) - 1U
+};
+
+#define TOPIC_BUFFER_SIZE               256U
```

Comment:
IMO, these should be in the header file
If these are not configurable then it should be marked as such
If these are configurable these should then be wrapped in `#ifndef`. This way users of the library can choose how long to set these to be

Reply:
Updated